### DISCUSSION TITLE
Bad Actors Among Us?

### DISCUSSION THREAD

**Discussion by Rob-NY** [jamulussoftware/jamulus#2999] [General]
There are individuals (one or more) participating in this community that seem to be purposefully causing both confusion in the directory listings space and invading servers for the purpose of surreptitiously recording sessions.  At the same time, this individual or individuals are contributing suggestions to code and documentation that are aimed to expressly prevent the very things they, themselves, are introducing into the ecosystem.

@pljones - there are other server operators out there that may have an actionable cause and I am not aware of a way to contact or petition the directory operators for resolution.  This could potentially be an issue as the core directory servers are bundled as "known good actors" in both the old and new privacy policies.  Is there a currently acceptable dispute resolution process that doesn't take place in this public forum?

**Comment by pljones** [comment]
To be frank, I've no idea what you're on about.

**Reply by Rob-NY** [reply]
@pljones - you may not understand the specifics, but there is a question in my post about how to contact directory operators (those that are part of the jamulus project itself) about specific or legal issues.  Is there a way?

**Reply by pljones** [reply]
Yes.  Post here.

**Reply by ann0see** [reply]
> Yes. Post here.

If it's confidential - what I think, it's best NOT to post it in public. Handling it [similar to a security vulnerability](https://github.com/jamulussoftware/jamulus/blob/main/SECURITY.md) should be the way to go IMHO.

**Reply by Rob-NY** [reply]
@ann0see  - thanks, I like that path better.   I am not the complainant in these matters, so I will pass along.

**Reply by mcfnord** [reply]
No confidentiality is merited. In fact, the opposite is needed here. There are no genuine legal issues, but there is a pattern of threats, abuse, bullying, and whisper campaigning. I took some time to size it up, and the pattern got incredibly clear.

The complainant is a narcissist abuser, and [Rob-NY is acting here as his "flying monkey"](https://psychcentral.com/pro/exhausted-woman/2019/07/narcissists-and-their-flying-monkeys#:~:text=When%20the%20narcissist%20wants%20to,assaults%2C%20threats%2C%20and%20violence.). I believe the complainant beat up a nice guy so the very upset guy would also act as a "flying monkey". It was the work of a cowardly, malevolent actor. A healthy, grown man would have approached me directly, but narcissists (and sociopaths) accrue human collateral damage to accomplish their objectives.

My software indicates that minute-for-minute, I am usually the most commonly connected user on the entire public network. At any hour, I'm the _most likely_ person out of everyone _globally_ to be connected to a public server. YET... Cant Be Found? But narcissists never just walk up to the victim and express concerns. Instead, [they practice "triangulation"](https://psychcentral.com/blog/psychology-self/2019/10/triangulation-and-narcissism#1). This innuendo complaint seeking secrecy is an exceptional example of triangulation. So tick **Triangulation** and **Flying Monkey** off your BINGO card.

I've identified a pattern of IP blocking from the complainant's servers. This is disappointing, and the blocks are very long-lasting. I don't want to direct people to an unstable version of this software, and that's what arbitrary IP banning results in. **How do Europe and Thailand thrive without this extreme degree of control?** People need access to servers that don't routinely IP-ban, so I've deployed seven servers across North America that do not IP-ban arbitrarily. I've also (mostly) erased the secretive mystery complainant's servers from my website, so my users won't experience instability from arbitrary IP-banning. I have hundreds of users now, and it becomes a serious issue that more than one have been IP-banned from a whole fleet of servers. I don't want my users to experience that.

Rob-NY, do you provide the code that empowers the secret complainant to IP-ban across a fleet? Do thrill-kill IP-bans represent the kind of server operator power you feel so strongly is lacking in this project? Unstable servers are antithetical to Jamulus and I do not want them predominant in North America. 

[Deflection is next](https://survivingnarcissism.tv/deflecting-narcissist-2/). I strongly believe sunshine is the antidote to this pattern of abusive, dishonest, malevolent deeds. I invite all the people whose feelings I may have hurt to come forward. But narcissists are never direct. Instead we have all this goofy innuendo and scheming for backroom power plays. Pathetic. Completely, thoroughly pathetic. I see entirely toxic and textbook-abusive people who aren't here to make anything great. Ok, what are they here to do?

**Reply by ann0see** [reply]
I think the team has no clue what happened. At least Peter and I don't seem to know anything about that matter – so I'm afraid that we can do much. 
Moreover, we are not professionally trained to resolve (personal ?) conflicts and can only help to some little extent. It's still good to have a neutral discussion. 

To everyone: Please remember to remain calm, professional and stay within the [Forum Etiquette](https://github.com/jamulussoftware/jamulus/discussions/1580)

**Reply by Rob-NY** [reply]
@mcfnord - I was willing to address this privately - but I'm glad you stepped into the spotlight.

So if someone does not act exactly as you believe they should act, or does not believe in the same societal mission as you do, then you resort to name calling, categorization, and revenge?  

Personally, I share your mission to make servers available without restriction as I, too, have numerous servers hosted openly throughout the US and Europe that have hosted thousands over musicians over the past few years.  What I do **NOT** do is impose my beliefs on others, and I respect and appreciate the reality that there may be many use-cases for using Jamulus that I may not understand, appreciate, or be invited into.

The point of my original post concerned your exacting "revenge" through deliberate manipulation of server names for the sole purpose of creating confusion in the minds of casual users.  This is not OK and someone who is as altruistic as you claim should know better.

Further, your **_Jamulus.live_** product which joins open servers and surreptitiously records participants without their knowledge or consent - and then makes those recording available globally - undermines your participation in the privacy discussion elsewhere here.  And then you call out people who might want to block this type of behavior?  Shame on you.

Lastly, I do not make code available to individuals.  My code contributions are all made through PR's or through links for evaluation in other discussion threads here in this repo.  While I'm flattered that you think my contributions are the only way to block bad actors, rest assured there are a lot of smart people in this space who knew how to do that long before I came along.

Some of the people you're attacking in your post have done more to promote and evangelize for Jamulus than anyone. Robust and vibrant communities have been built and are open to everyone as long as they play by a common sense set of rules.  I believe you may just be angry because you've been excluded from these communities based on your behaviors there; and now you're seeking to retaliate.

Remember that operators are paying for servers with real money and real effort.  Each operator's motivations will be different and you should respect those choices.

**Reply by npostavs** [reply]
So is the basic complaint here that mcfnord is making recordings from Jamulus servers available on his jamulus.live website and you feel that is a prvacy violation or something like that?

**Comment by ann0see** [comment]
Corrados and pljones host the directories, gilgongo the domain. Social misbehavior in the Jamulus network can't be handled by the Jamulus team as we don't have access to all servers.

If there's anything confidential, contact the team E-Mail.

**Comment by ghost** [comment]
What is the motive here? I think it is bad motive by crackers. On the other hand, Jamulus gets me excited to jam. One may find strangers to jam with in public, then possibly go private. This is a different motive... Jamulus for musicians, not crackers. The public aspect is to find friends, the private aspect is to practice with friends. A band practices in private and performs in-person on the stage in public, Jamulus should be designed to accommodate musicians.
